package.json

```diff
@@ -64,19 +64,19 @@
     "@rollup/plugin-replace": "^2.0.0",
     "@types/react": "^16.0.0",
     "@types/react-dom": "^16.0.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.1.1",
```

Comment:
it sounds like the upstream version is in progress https://github.com/enzymejs/enzyme/pull/2430

Reply:
From reading some more comments there, it could take longer